Domains/Profiles/Profiles.ecschema.xml

```diff
@@ -366,4 +367,118 @@
         <ECProperty propertyName="Width" typeName="double" description="Extent of the capsule in the direction of the x-axis." category="ProfileProperties" kindOfQuantity="AECU:LENGTH_SHORT" />
         <ECProperty propertyName="Depth" typeName="double" description="Extent of the capsule in the direction of the y-axis." category="ProfileProperties" kindOfQuantity="AECU:LENGTH_SHORT" />
     </ECEntityClass>
+
+    <ECEntityClass typeName="ProfiledSolidDefinitionElement" modifier="Abstract" displayLabel="Profiled Solid Definition Element" description="TODO">
+        <BaseClass>bis:DefinitionElement</BaseClass>
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="PhysicalTypeUsesProfiledSolidDefinition" modifier="None" strength="referencing" strengthDirection="forward" description="TODO">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses" polymorphic="true">
+            <Class class="bis:PhysicalType"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is used by" polymorphic="true">
+            <Class class="ProfiledSolidDefinitionElement"/>
+        </Target>
+        <ECProperty propertyName="IsMirrored" typeName="boolean" displayLabel="Is Mirrored" description="TODO" />
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="PhysicalElementUsesProfiledSolidDefinition" modifier="None" strength="referencing" strengthDirection="forward" description="TODO">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses" polymorphic="true">
+            <Class class="bis:PhysicalElement"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is used by" polymorphic="true">
+            <Class class="ProfiledSolidDefinitionElement"/>
+        </Target>
+        <ECProperty propertyName="IsMirrored" typeName="boolean" displayLabel="Is Mirrored" description="TODO" />
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="IProfiledSolidDefinition" modifier="Abstract" displayLabel="IProfiled Solid Definition" description="TODO">
```

Comment:
Is IProfiledSolidDefinition required?  think it should be possible to replace it with ProfiledSolidDefinitionElement

Reply:
It is required to filter out the classes that can be used as "Source" of the "Uses" relationships. e.g.
![image](https://user-images.githubusercontent.com/46930149/144070888-bd566463-7a1d-4845-8210-c343b02b6596.png)

Reply:
That is, not all ProfiledSolidDefinitionElement-subclasses can be sources of such "Uses" relationships. E.g. SegmentedProfiledSolidDefinition cannot be a source in them.